Application/Frontend/accordTypes.ts

```diff
@@ -37,4 +41,13 @@ export interface IconProps {
 
 export interface NewChatModalProps extends NewFriendModalProps {
   onCreateChat: (recipients: string[]) => void;
```

Comment:
- Add error handling for potential errors during the execution of onCreateChat callback. For example, if the callback fails to create a chat session, provide feedback to the user with an error message or notification.

Reply:
I agree! Once we are done will all of our backend API endpoints, we can look through all of them and ensure that we have better error handling.